winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+
+### Excel version
+
+While exporting to Excel, you can specify the excel version by using `ExcelVersion` property.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting stacked headers
+
+You can export stacked headers to excel by setting `CanExportStackedHeaders` property to `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportStackedHeaders = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting merged cells
+
+You can export merged cells to excel by setting `CanExportMergedCells` property as `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportMergedCells = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting unbound rows
+
+You can export unbound rows to excel by setting `CanExportUnboundRows` property as `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportUnboundRows = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Changing start row and column index while exporting
+
+You can export the data to specified row index and column index in worksheet, by setting `StartRowIndex` and `StartColumnIndex` properties.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.StartColumnIndex = 3;
+options.StartRowIndex = 3;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-with-row-column-index.png" alt="Exporting to Excel with Custom Row and Column Index in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Export DataGrid SelectedItems to Excel
+
+By default, entire grid will be exported to Excel. You can export selected rows only by passing `SelectedItems` to 
```

Comment:
By default, the entire grid will be exported to Excel. Export selected rows only by passing the `SelectedItems` to

Reply:
Content modified